dashboard/cron.py

```diff
@@ -80,6 +80,37 @@ class DashboardCronJob(CronJobBase):
     schedule = Schedule(run_at_times=settings.RUN_AT_TIMES)
     code = 'dashboard.DashboardCronJob'    # a unique code
 
+    # return a list of invalid course ids
+    def invalid_course_ids(self):
+
+        # whether all course ids are valid ids
+        invalid_course_id_list = []
```

Comment:
I'd probably make this an attribute in the class so it could be used by the other methods later. We could alter this to have the other methods just skip the invalid courses rather than stop the job entirely.

Reply:
This is probably fine that it is for now, that's an easy change later. I'm thinking this is the "good" solution. The better solution would be if this could continue to process the good courses, but I wouldn't worry about that for now.